packages/calcite-components/src/components/chip/chip.tsx

```diff
@@ -221,6 +221,7 @@ export class Chip
     if (event.target === this.el) {
       switch (event.key) {
         case " ":
+          event.stopPropagation();
```

Comment:
Is it possible to handle this within the Combobox? Maybe in `keyDownHandler` by checking to see if the event.target is not a Chip, or is only the input element, etc.?